main.js

```diff
@@ -1,30 +1,47 @@
-const studentList =[];
-const studentName = document.querySelector('#studentName');
-const houses = [
-  'Gryffindor', 
-  'Hufflepuff', 
-  'Ravenclaw', 
-  'Slytherin'
-];
-const startBtn = document.querySelector('#startBtn');
-const sortBtn = document.querySelector('#sortBtn');
-
+const studentList = [];
+const houses = ["Gryffindor", "Hufflepuff", "Ravenclaw", "Slytherin"];
+const startBtn = document.querySelector("#startBtn");
```

Comment:
Let's move lines 3 and 4 into the buttonClick function or another function that you call immediately on app load. We don't want any actions taking place outside of function.

Also, for the houses, unless you are using them in multiple places, let's put them in the function that is accessing them so that they are out of global scope.